packages/plugin-react/src/index.js

```diff
@@ -15,6 +15,11 @@ module.exports = class BugsnagReactPlugin {
           error: null,
           info: null
         }
+        this.handleClearError = this.handleClearError.bind(this)
+      }
+
+      handleClearError () {
+        this.setState({ error: null, info: null })
```

Comment:
did you think better of making this async to prevent synchronous loops in the end?

Reply:
I added a test case and tried it out on a real app and it doesn't result in a stack overflow. The error repeats a handful of times then stops so I assume React has its own detection mechanism and prevents an infinite rendering loop.